### PR TITLE
New version: gerardog.gsudo version 2.1.0

### PR DESCRIPTION
**Pull request by gerardog**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111683&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/g/gerardog/gsudo/2.1.0/gerardog.gsudo.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ReleaseDate: 2023-07-09
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/gerardog/gsudo/releases/download/v2.1.0/gsudo.setup.x86.msi
+  InstallerSha256: 4BBC4ECC8ABA137958B2CF683E9627ABEE9DAACD265F98256D137051B34B40D7
+  ProductCode: '{3DBF0D3C-BA18-44CF-AE40-5C0AEA81C60F}'
+- Architecture: x64
+  InstallerUrl: https://github.com/gerardog/gsudo/releases/download/v2.1.0/gsudo.setup.x64.msi
+  InstallerSha256: 3BA8A3F79AA5408944797038EE57D93D1E331A77B960A2B0F3BB2E0B7A18D9E3
+  ProductCode: '{E918BCF9-EED2-465A-862C-F5BFFC25F82A}'
```
Comment (mdanish-kh) [inline manifests/g/gerardog/gsudo/2.1.0/gerardog.gsudo.installer.yaml:24]:
```suggestion
  ProductCode: '{6296AF88-0AF9-4DDA-B285-88D7FC75E2C7}'
```

File: manifests/g/gerardog/gsudo/2.1.0/gerardog.gsudo.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+UpgradeBehavior: install
+Commands:
+- gsudo
+- sudo
+ReleaseDate: 2023-07-09
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/gerardog/gsudo/releases/download/v2.1.0/gsudo.setup.x86.msi
+  InstallerSha256: 4BBC4ECC8ABA137958B2CF683E9627ABEE9DAACD265F98256D137051B34B40D7
+  ProductCode: '{3DBF0D3C-BA18-44CF-AE40-5C0AEA81C60F}'
```
Comment (mdanish-kh) [inline manifests/g/gerardog/gsudo/2.1.0/gerardog.gsudo.installer.yaml:20]:
```suggestion
  ProductCode: '{9C94B186-FA68-4EF0-A8AA-444356D6D7D8}'
```

File: manifests/g/gerardog/gsudo/2.1.0/gerardog.gsudo.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- Architecture: x64
+  InstallerUrl: https://github.com/gerardog/gsudo/releases/download/v2.1.0/gsudo.setup.x64.msi
+  InstallerSha256: 3BA8A3F79AA5408944797038EE57D93D1E331A77B960A2B0F3BB2E0B7A18D9E3
+  ProductCode: '{E918BCF9-EED2-465A-862C-F5BFFC25F82A}'
+- Architecture: arm64
+  InstallerUrl: https://github.com/gerardog/gsudo/releases/download/v2.1.0/gsudo.setup.arm64.msi
+  InstallerSha256: AF00F648042DB2331ECD6DD67C8E4CB996700CE872CE251609306E68B45F3E10
+  ProductCode: '{0253A814-004A-42BA-A30E-D4AFCC8B3DF2}'
```
Comment (mdanish-kh) [inline manifests/g/gerardog/gsudo/2.1.0/gerardog.gsudo.installer.yaml:28]:
```suggestion
  ProductCode: '{E3F18403-C296-4E8F-96AA-4163C986D8FE}'
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Update ProductCodes

File: manifests/g/gerardog/gsudo/2.1.0/gerardog.gsudo.installer.yaml (hunk 1 above)
Comment (gerardog) [reply manifests/g/gerardog/gsudo/2.1.0/gerardog.gsudo.installer.yaml]: Hi @vedantmgoyal2009 do you have any idea why winget releaser v2 may have used different ProductCodes this time?
https://github.com/gerardog/gsudo/actions/runs/5497391130/jobs/10018145060#step:7:1

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**